### PR TITLE
new icon: influxdb (original, original-wordmark)

### LINKED ISSUE #1148: [ICON REQUEST]: influxDB
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

InfluxDB is the time series platform for developers

### Links and sources

* <a href="https://www.influxdata.com/products/influxdb-overview/">website</a>
* <a href="https://github.com/influxdata/influxdb">github</a>
* <a href="https://influxdata.github.io/branding/logo/downloads/">svg</a>

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Panquesito7**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [ ] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1148**

## Link to prove your SVG is correct and up-to-date.

https://www.influxdata.com/

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Devicon.json looks good, however the wrong icons have been used for the product 🙈

These are the company logos (influxdata) and not the product logos (influxdb). The product logos can also be downloaded from their website, here: https://influxdata.github.io/branding/logo/brands/

Please change to the correct icons and let me kn ow once it's done 😄

File: icons/influxdb/influxdb-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....082 1.71-1.886.43-.809.645-1.739.645-2.79 0-1.054-.214-1.984-.644-2.788-.414-.809-.984-1.438-1.711-1.887-.73-.453-1.582-.68-2.555-.68-.953 0-1.804.227-2.55.68-.731.45-1.31 1.078-1.739 1.887-.414.804-.617 1.734-.617 2.789 0 1.05.203 1.98.617 2.789.43.804 1.008 1.433 1.738 1.886.747.45 1.598.676 2.551.676Zm19.637 2.14c-1.102 0-2.102-.262-3-.79-.879-.527-1.57-1.269-2.074-2.226l-.059 2.567h-2.328v-22.82h2.387v11.296c.558-.957 1.297-1.7 2.215-2.226.914-.524 1.96-.79 3.14-.79 1.309 0 2.469.32 3.477.961 1.011.637 1.804 1.52 2.386 2.645.579 1.129.868 2.426.868 3.89 0 1.387-.317 2.657-.953 3.801a7.31 7.31 0 0 1-2.524 2.707c-1.047.657-2.226.985-3.535.985Zm-.336-2.14c.973 0 1.824-.227 2.555-.677a4.706 4.706 0 0 0 1.71-1.886c.43-.809.641-1.739.641-2.79 0-1.054-.21-1.984-.64-2.788-.414-.809-.985-1.438-1.711-1.887-.73-.453-1.582-.68-2.555-.68-.953 0-1.805.227-2.55.68-.731.45-1.31 1.078-1.739 1.887-.414.804-.617 1.734-.617 2.789 0 1.05.203 1.98.617 2.789.43.804 1.008 1.433 1.738 1.886.746.45 1.598.676 2.551.676Zm10.328-17.009v-2.996h-1.11v-.593h2.9v.593h-1.102v2.996Zm2.406 0 .184-3.59h.664l1.09 2.282 1.113-2.281h.664l.184 3.59h-.61l-.113-2.657-.984 2.035h-.485l-.992-2.054-.11 2.675Zm0 0"/></svg>
```
Comment (Snailedlt) [inline icons/influxdb/influxdb-original-wordmark.svg:1]: Is it possible to combine the paths here?

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: icons/influxdb/influxdb-original-wordmark.svg (hunk 1 above)
Comment (Panquesito7) [reply icons/influxdb/influxdb-original-wordmark.svg:1]: I'll see if I can do that. Thanks. 🙂

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark: 

Thanks for another great contribution @Panquesito7 :100:

**Pull request merged**